x509/src/fmc_alias_cert.rs

```diff
@@ -13,7 +13,205 @@ Abstract:
 --*/
 
 // Note: All the necessary code is auto generated
-include!(concat!(env!("OUT_DIR"), "/fmc_alias_cert_tbs.rs"));
+pub struct FmcAliasCertTbsParams<'a> {
```

Comment:
Let's still check these in in separate files (can even keep the names). That way it'll be easier to regenerate them.

Reply:
Done